ovh-api-bash-client.sh

```diff
@@ -201,28 +229,70 @@ getJSONFieldString()
 {
     JSON="$1"
     FIELD="$2"
-    RESULT=$(echo $JSON | $CURRENT_PATH/$LIBS/JSON.sh | grep "\[\"$FIELD\"\]" | sed -r "s/\[\"$FIELD\"\]\s+(.*)/\1/")
+    RESULT=$(echo $JSON | $BASE_PATH/$LIBS/JSON.sh | grep "\[\"$FIELD\"\]" | sed -r "s/\[\"$FIELD\"\]\s+(.*)/\1/")
     echo ${RESULT:1:${#RESULT}-2}
 }
 
+# override CURRENT_PATH with profile name
+# usage : initProfile profile_name
+initProfile()
+{
+  local profile=$1
+  if [ -n "${profile}" ]; then
+
+    if [ ! -d "${PROFILES_PATH}/${profile}" ]
+    then
+     echo "${PROFILES_PATH}/${profile} should exists"
+     exit 1
+    fi
+    # override default configuration location
+    CURRENT_PATH="$( cd "${PROFILES_PATH}/${profile}" && pwd )"
+    HELP_CMD="${HELP_CMD} --profile ${profile}"
+  else
+    echo "profile not set, choose in : "
```

Comment:
@denouche this doesn't break users : if --profile is not set, the keys files goes to script location. 
(the CURRENT_PATH in only overrided in the initProfile function)

I can :
- simulate an explicit profile _default_  : if setted to that value, or without --profile xxx
- moving the default location (CURRENT_PATH value) to the initProfile function.

It should be more logical.